subsys/net/ip/l2/Kconfig

```diff
@@ -9,7 +9,7 @@ menu "Link layer options"
 config NET_L2_DUMMY
 	bool "Enable dummy l2 layer"
 	default n
-	default y if !NET_L2_ETHERNET && NET_SLIP_TUN
+	default y if !NET_L2_ETHERNET
```

Comment:
This is incorrect now, the dummy driver is used in tests/net so we could say "default y if !NET_L2_ETHERNET && NET_TEST" here